modules/scripts-utils/dtbook-utils/src/main/resources/xml/fix-dtbook/fix-dtbook.xpl

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you include some documentation to explain that this code came from Pipeline 1 originally?

Reply:
@NPavie it would be nice to have a comment here that the module was ported from Pipeline 1, and with a mentioning of both you and the original authors.

Reply:
@NPavie I don't see Pipeline 1 and the original authors mentioned anywhere yet.

Reply:
(sorry i just see now the comment you made in the review tab)

Noted, will do

Reply:
I got back the list of authors that have participated to the dtbook fix script.
I will add a comment stating this script is a portage of pipeline 1 se_tpb_dtbookFix transformer followed by the list of authors in order of participation.

Reply:
should be good in 59df9d0bd3f5fa95cfde4dbc632daaa5af352eed

Reply:
I'll also add a similar mention with list of authors in other pipeline 1 ported scripts

Reply:
Thanks!